src/app/components/ArticleMetadata/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { string, shape, arrayOf } from 'prop-types';
+
+const renderTags = tags =>
+  tags.map(({ thingLabel: content }) => (
+    <meta name="article:tag" content={content} key={content} />
+  ));
+
+const ArticleMetadata = ({
+  author,
+  firstPublished,
+  lastPublished,
+  section,
+  aboutTags,
+  mentionsTags,
+}) => (
+  <Helmet>
+    <meta name="article:author" content={author} />
+    <meta name="article:modified_time" content={lastPublished} />
+    <meta name="article:published_time" content={firstPublished} />
+    {section && <meta name="article:section" content={section} />}
+    {Boolean(aboutTags && aboutTags.length) && renderTags(aboutTags)}
+    {Boolean(mentionsTags && mentionsTags.length) && renderTags(mentionsTags)}
+  </Helmet>
+);
+
+const tagPropTypes = shape({
```

Comment:
Should this not go in the proptype folder?

Reply:
I really think we should keep the proptypes inside of the component instead of being in an entirely different location.

Reply:
If the prop types are not shareable i.e. isolated to this component/container I'd keep them here.